aiida/cmdline/commands/cmd_devel.py

```diff
@@ -61,8 +64,16 @@ def get_valid_test_paths():
 @decorators.with_dbenv()
 def devel_run_daemon():
     """Run a daemon instance in the current interpreter."""
-    from aiida.engine.daemon.runner import start_daemon
-    start_daemon()
+    try:
+        # Keep the import also within the try-except such that if the import itself excepts, it is also caught
+        from aiida.engine.daemon.runner import start_daemon
```

Comment:
sorry for the nit-picking but what happens if the import here excepts?
wouldn't that be handled properly anyhow, i.e. wouldn't the exception appear in the regular way?

Reply:
No, it would have the same problem. This verdi command is what is launched by circus in a different sub process. If that sub process excepts before the daemon logging handling is configured (which is what happens in the `aiida.engine.daemon.runner.start_daemon` call, the exception that will be raised will never end up in the log file and so the exception will never be seen by the user.